File: Java_Programming/Competative/Assignments/Assignment_2/program3.java
package Competative.Assignments.Assignment_2;

//Accept one number from user if number is less than 10 then print Hello otherwise print Demo

import java.util.*;

class DisplayX 
{

    public void Display(int iNo)
    {
        if(iNo < 10)
        {
            System.out.println("Hello");
        }
        else
        {
            System.out.println("Demo");
        }
    }
}

class program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number");
        iValue = sobj.nextInt();

        DisplayX dobj = new DisplayX();

        dobj.Display(iValue);

    }
}

File: Cpp_Programming/Competative/Assignments/Assignment_2/program3.cpp
//Accept one number from user if number is less than 10 then print Hello otherwise print Demo

#include<iostream>
using namespace std;

int Display(int iNo)
{
    if(iNo < 10)
    {
        cout<<"Hello";
    }
    else
    {
        cout<<"Demo";
    }
}

int main()
{
    int iValue = 0;

    cout<<"Enter the number\n";
    cin>>iValue;

    Display(iValue);

    return 0;
}